bin/vendor-licenses

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Usage:
+
+    ./bin/vendor-licenses > identify/vendor/licenses.py
+"""
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
could remove these imports + exclude this from the hook i guess /shrug

Reply:
yeah, figured it doesn't hurt -- it's syntax compatible with python2, just not runtime compatible